src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -248,19 +275,68 @@ public Config(IGerritHudsonTriggerConfig config) {
         enableProjectAutoCompletion = config.isEnableProjectAutoCompletion();
         projectListFetchDelay = config.getProjectListFetchDelay();
         projectListRefreshInterval = config.getProjectListRefreshInterval();
-        if (config.getCategories() != null) {
-            categories = new LinkedList<VerdictCategory>();
-            for (VerdictCategory cat : config.getCategories()) {
-                categories.add(new VerdictCategory(cat.getVerdictValue(), cat.getVerdictDescription()));
-            }
-        }
+
+        assertDefaultCategories();
+
         if (config.getReplicationConfig() != null) {
             replicationConfig = new ReplicationConfig(config.getReplicationConfig());
         }
         watchdogTimeoutMinutes = config.getWatchdogTimeoutMinutes();
         watchTimeExceptionData = addWatchTimeExceptionData(config.getExceptionData());
     }
 
+    public static Integer getIntegerFromString(String str) {
+        if (str != null && !str.equals(""))
+            return Integer.parseInt(str);
```

Comment:
Are you sure this won't throw `NumberFormatException`?

Reply:
Well, I was thinking of propagating the exception to the UI. I've added a "StringUtils.isNumeric(str)" in the if statement.

Reply:
StringUtils.isNumeric("-1") seems to return false, so I used NumberUtils.isNumeric instead